smoke-test/tests/authenticated/teaspoons_pipeline_runs_list_tests.py

```diff
@@ -0,0 +1,11 @@
+from ..teaspoons_smoke_test_case import TeaspoonsSmokeTestCase
+
+
+class TeaspoonsPipelineRunsListTests(TeaspoonsSmokeTestCase):
```

Comment:
these are really quite clear, but it would make it even easier to quickly understand what we're testing if we add a docstring explaining what gets tested in each class of tests